database/migrations/2024_05_13_204255_modify_sites_polygons.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        if (! Schema::hasTable('site_polygon')) {
+            return;
+        }
```

Comment:
I suppose this check isn't hurting anything, but Laravel should guarantee that it always passes because the migrations get run in order.

Reply:
This was just to ensure that someone running the migration independently wouldn't encounter issues. However, as you mentioned, there shouldn't be any problems with the execution since it proceeds in order.